### ISSUE TITLE
sv.MulticlassOversampling error for getattr() function

### ISSUE THREAD

**Issue by azamkamranian** [analyticalmindsltd/smote_variants#75]
I am checking documents example of package. The following example gave me the error 'TypeError: getattr(): attribute name must be string'. Why? 

'''
import smote_variants as sv import sklearn.datasets as datasets dataset= datasets.load_wine() 
oversampler= sv.MulticlassOversampling(sv.distance_SMOTE) X_samp, y_samp= oversampler.sample(dataset['data'], dataset['target'])
'''

**Comment by gykovacs** [comment]
Thank you for raising, I look into it.

**Comment by gykovacs** [comment]
Correct, there was a minor change in the interface recently and the docs have not been updated accordingly. Now I updated, it should be out with the next release (in a couple of days).

The correct use:
```Python
oversampler = sv.MulticlassOversampling(oversampler='distance_SMOTE', oversampler_params={})
```

You can specify any parametrization through the ``oversampler_params`` argument.
